### DISCUSSION TITLE
Importing projects without opening editor

### DISCUSSION THREAD

**Discussion by SuperMidas-01** [godotengine/godot-proposals#10306] [General Discussions]
This is pretty minor, but I rearranged my project organization and had to reimport each one in the project manager. After selecting a file, my options were to 'Import & Edit" or "Cancel", which meant I had to open the editor and reopen the project manager for each project. I think there should be an "Import" only button. You could probably get rid of the "Import & Edit" since editing is a double click away after importing anyway. I'd open an issue, but I'm pretty clueless with programming outside of scripting.

Edit: Just realized that "Scan" will do this for me, but this change could still be of use in general.

**Comment by AThousandShips** [comment]
You can also do this with the command line, see [here](https://docs.godotengine.org/en/latest/tutorials/editor/command_line_tutorial.html)
